config/zones.json

```diff
@@ -642,7 +644,7 @@
       "coal": 9535,
       "gas": 30723,
       "geothermal": 0,
-      "hydro": 14614,
+      "hydro": 20281,
```

Comment:
Where did you get this new figure (+6GW compared to previous)? I think (although i'm not sure) that I had chosed to used IRENA data cause spain entose hydro capacity figure was incorrect (see readme)

Reply:
@brunolajoie 
Yeah, I was about to check the readme for sources and any different sources than ENTSO-E. Then github was offline for a few minutes and I simply forgot about it. 
For Spain, I just saw the annotations below the chart on entso-e.
Pumped consumption capacity is included in "Pumped storage", but generating capacity is assigned to "water reservoir" ;) I guess that's why we have ~+5.6 GW??!?! 
It says on entso-e: 1156 MW run-of-river and 19125 MW hydro reservoir (which is including about 5600 MW pumped storage gen capacity which might not be included in IRENA (?) data or wherever. By the way: in the readme it says ENTSO-E for Spain?)

![image](https://user-images.githubusercontent.com/25743609/34845650-ebd42934-f715-11e7-8b0a-3afe72a298a3.png)

I am confused now. :D Do I keep the new value for pumped storage as it is so that consumption is shown as ratio of installed capacity? Or do I better leave it blank?

Reply:
and: I will check and update the readme tomorrow!

Reply:
Readme says Spain hydro capacity is currently IRENA.  Unless proven wrong, I'd rather keep using IRENA as long as these confusing comments from ENTSEO are still there xD

Reply:
IRENA is mentioned in the "real time generation"-part for Spain and not in "installed capacity"-part. That's why I had overseen it there. I will fix this one, too, and copy it to capacities ;)

Reply:
@brunolajoie Maybe you can take another look at the data on IRENA for Spain.
It also has Pumped storage (you have to select it manually)

![image](https://user-images.githubusercontent.com/25743609/34873987-775413ee-f797-11e7-84cf-d9b05cc37e1c.png)

I also couln't  manage to calculate the currently installed "14614 MW" using the numbers from IRENA, maybe there is another error somewhere?

Latest available data for 2016 from IRENA for Spain:
Pumped storage: 3 280 MW
Large hydro: 12 133 MW
Medium hydro: 1 673
Small hydro: 280 MW
Mixed hydro: 2 687 MW

Sum of all hydro: 20 053 MW which is very close to ENTSO-E numbers.

Reply:
IRENA figures looks good then! But on your PR, hydro+storage tops up 25GW so it was a bit too high